build/Microsoft.DotNet.Cli.Run.targets

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
The `clean.targets` file only has code to clean NuGet cache locations, so it is not sufficient